Dockerfile

```diff
@@ -104,12 +112,24 @@ ENV OCM_URL="https://api.github.com/repos/${OCM_URL_SLUG}/releases/${OCM_VERSION
 # ocm is not in a tarball
 RUN mkdir /ocm
 WORKDIR /ocm
+
+## arm-native
+# Download the checksum
+RUN [[ $(platform_convert "@@PLATFORM@@" --x86_64 --arm64) != "arm64" ]] && exit 0 || /bin/bash -c "curl -sSLf $(curl -sSLf ${OCM_URL} -o - | jq -r '.assets[] | select(.name|test("linux-arm64.sha256")) | .browser_download_url') -o sha256sum.txt"
```

Comment:
I'm not a _huge_ fan of the way this is done, but these quotes inside single quotes inside quotes just absolutely _destroy_ any chance of shell expansion :(